omise.js/form-example/index.html

```diff
@@ -66,21 +67,18 @@
 
       <div class="form-group">
         <label>Security code</label>
-        <input type="text" name="securityCode" class="form-control" placeholder="123" />
+        <input type="text" data-name="securityCode" class="form-control" placeholder="123" />
       </div>
 
       <div class="form-group">
         <button class="btn btn-primary">Checkout</button>
       </div>
-    </form>
-
 
-    <form id="checkout-form" action="/checkout" method="POST">
       <input type="hidden" name="token" />
```

Comment:
@ratchagarn just minor but can we rename this field to `omiseToken` instead?
Because our example in the Omise document will say that backend will retrieve a token through
```php
$charge = OmiseCharge::create([
    'card' => $_POST['omiseToken']
]);
```
That we can use above code as an example on both omise.js use cases (manual implement omise.js / using built-in card form cases) without any confusion.

🙇